app/view-plants/page.tsx

```diff
@@ -0,0 +1,110 @@
+'use client';
+
+import { useEffect, useState } from 'react';
+import { UUID } from 'crypto';
+import supabase from '@/api/supabase/createClient';
+import { get_plant_by_id } from '@/api/supabase/queries/plant_by_id';
+import { getPlantSeasonality } from '@/api/supabase/queries/plantSeasonality';
+import PlantCard from '@/components/PlantCard/PlantCard';
+import { Plant } from '@/types/schema';
+
+export default function Home() {
+  const [selectedOption, setSelectedOption] = useState<'myPlants' | 'all'>(
+    'myPlants',
+  );
+  const [isSelected, setIsSelected] = useState<true | false>(false);
```

Comment:
you can do useState<boolean>(false)!

Reply:
nit: maybe we can rename these states to 
`viewingOption` / `setViewingOption` and `inAddMode` / `setInAddMode` respectively for clarity